source/transactions/tests/README.rst

```diff
@@ -360,13 +364,17 @@ instead.
       @require_mongos_count_at_least(2)
       def test_unpin_for_non_transaction_operation(self):
         client = MongoClient(mongos_hosts)
+        # Wait until more than one mongos is discovered.
+        wait_until(lambda: len(client.nodes) > 1)
```

Comment:
You don't think we also need to bump the latency threshold?

Reply:
Done.